src/PanoViewer/PanoViewer.js

```diff
@@ -570,47 +554,26 @@ export default class PanoViewer extends Component {
 		}
 	}
 
-	/**
-	 * Create webgl context and initiate user interaction and rendering
-	 * @ko WebGl 컨텍스트 생성 및 사용자 상호 작용과 렌더링 시작
-	 * @method eg.view360.PanoViewer#resume
-	 */
-	resume() {
-		if (this._isResumed || !this._yawPitchControl) {
-			return;
-		}
-
-		if (!this._photoSphereRenderer) {
-			this._initRenderer(this._yaw, this._pitch, this._fov, this._projectionType);
-		}
-
-		// do setTimeout for not blocking UI when resume is called in syncrounos code.
-		setTimeout(() => {
-			if (this._isResumed || !this._photoSphereRenderer) {
-				return;
-			}
-
-			this._photoSphereRenderer
-				.bindTexture()
-				.then(() => this._resume())
-				.catch(() => {
-					this._triggerEvent(EVENTS.ERROR, {
-						type: ERROR_TYPE.FAIL_BIND_TEXTURE,
-						message: "failed to bind texture"
-					});
+	_bindTexture() {
+		this._photoSphereRenderer
+			.bindTexture()
+			.then(() => this._activateInteraction())
+			.catch(() => {
+				this._triggerEvent(EVENTS.ERROR, {
+					type: ERROR_TYPE.FAIL_BIND_TEXTURE,
+					message: "failed to bind texture"
 				});
-		}, 0);
+			});
 	}
 
-	_resume() {
+	_activateInteraction() {
```

Comment:
I think naming is narrow than it's functionality.

It activates interaction but also activates renderer.

Reply:
I agree with you. will change _activateInteraction/_inactivateInteraction to _activate/_deactivate pair.